src/sagemaker/session.py

```diff
@@ -280,7 +277,6 @@ def download_data(self, path, bucket, key_prefix="", extra_args=None):
                 https://boto3.amazonaws.com/v1/documentation/api/latest/guide/s3-example-download-file.html
 
         Returns:
-
```

Comment:
i think you should not have a `Returns:` section if you're not going to put anything in it, or have:

```
Returns:
    None
```

Reply:
ah that's understandable. I didn't remove it.